code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -653,6 +653,26 @@
 					message = "<span class='danger'>slaps [T.himself]!</span>"
 					playsound(loc, 'sound/effects/snap.ogg', 50, 1)
 
+// Citadel changes starts here
+		if("aslap", "aslaps")
+			m_type = 1
+			if(!restrained())
+				var/M = null
+				if(param)
+					for(var/mob/A in view(1, null))
+						if(param == A.name)
+							M = A
+							break
+				if(M)
+					message = "<span class='danger'>slaps [M]'s butt. Oof!</span>"
```

Comment:
You can remove the `Oof!`, and leave just the `slaps X's butt!`

Reply:
It was inspired by *slap having `Ouch!` at the end BUT OKAY I'LL REMOVE THIS.